api/tests/test_rate_limit.py

```diff
@@ -168,20 +257,69 @@ async def test_acquire_multiple_servers(rate_limiter_factory: RateLimiterFactory
         pytest.lazy_fixture('memory_rate_limiter_factory'),
     ],
 )
-async def test_release_multiple_servers(rate_limiter_factory: RateLimiterFactory):
+async def test_room_limit_multiple_servers(rate_limiter_factory: RateLimiterFactory):
+    server_1 = await rate_limiter_factory('server-id-1')
+    server_2 = await rate_limiter_factory('server-id-2')
+
+    # Acquire the maximum number of room slots across two servers
+    for i in range(0, MAX_CONNECTIONS_PER_ROOM - 1):
+        await server_1.acquire_connection(f'user-{i}', 'room-1')
+    await server_2.acquire_connection('user-1', 'room-1')
```

Comment:
This looks like it should not work. user-1 is the second user to connect to the room. When user-1 connects again on server-2, I would expect there to still be an open room slot (unless we want a user to be able to hold 2 slots?)

Reply:
A user can hold two slots because they can have multiple browser tabs (Or multiple computers on the same network)